docker.go

```diff
@@ -255,6 +257,19 @@ func (docker *Docker) Host() *Docker {
 	return docker
 }
 
+// ExposeContainer sets the container to expose the following TCP ports
+//		ports:			the TCP ports the container is exposing
+func (container *Container) ExposeContainer(ports ...int) *Container {
```

Comment:
I don't think we should have the `Container` part in the `ExposeContainer` method name: The method is already defined on a `Container`, so users should be able to infer the type easily (and we can't conflict).

I'm also thinking it could make it easier for people to transition from Marathon pre-v1.5 to v1.5+ if most method names stay the same. We might even reuse part of the logic. WDYT?

(Similar argumentation for other new methods also coming with a `Container` name part.)